### ISSUE TITLE
Training and test set split.

### ISSUE THREAD

**Issue by JasonMilesCS** [autonomousvision/differentiable_volumetric_rendering#54]
Hi @m-niemeyer , Thanks for your great work and clean code.  I noticed that you uploaded a pre-trained model on all categories. But the dataset split for this pre-trained model is unavailable. Just wondering whether you may release it?

**Comment by m-niemeyer** [comment]
Hi @JasonMilesCS , thanks a lot for your interest in our project!

If you [download the datasets](https://github.com/autonomousvision/differentiable_volumetric_rendering#download-datasets), you will find the files "train.lst", "val.lst", "test.lst" in each class folder; these are the train/val/test splits for each class.  I hope this helps a little!

**Comment by JasonMilesCS** [comment]
I downloaded this several times but failed since it is too big... I will try it again. Thanks for your reply.
